theme/less/_navigation.less

```diff
@@ -1,25 +1,103 @@
-nav
+// The main navigation menu
+
+#main-navigation 
 {
-  ul
+  background: linear-gradient(#fcfcfc, #ececec);
+}
+
+#main-navigation > ul
+{
+  height:50px;
+  display: flex;
+  padding: 0;
+  border-color: #EAEAEA;
+  border-radius: 5px;
+  border-style: solid;
+  border-width: 1px;
+  box-shadow: 0 1px 1px rgba(0, 0, 0, 0.2);
+}
+
+#main-navigation > ul li
+{
+  padding: 15px;
+  position: relative;
+  list-style: none;
+  border-right: 1px solid #e3e3e3;    
+}
+
+#main-navigation > ul ul
+{
+  top: 50px;
+  display: block;
+  position: absolute;
+  left: 0;
+  padding: 0px;
+  background-color: #f0f0f0;
+}
+
+#main-navigation > ul ul ul
+{
+  left: 99%;
+  top: 0;
+  margin-left: 1px;
+}
+
+#main-navigation > ul li:not(:hover) > ul
+{
+  display:none;
+}
+
+#main-navigation ul ul li
+{
+  padding:10px;
+  min-width: 150px;
+}
+
+#main-navigation ul ul li:hover, ul ul a:hover
+{
+  background-color: #ffffff;
+}
+
+#main-navigation ul ul a
+{
+  text-decoration: none;
+  color: #595959;
+}
+
+#main-navigation > ul > li > a
+{
+  text-decoration: none;
+  font-weight: bold;
+  font-family: "PT Sans", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
+  font-size: 14px;
+  color:#2883DE;
+  text-shadow: 1px 1px 0px #fff;
+}
+
+#main-navigation ul li img
+{
+  float: right;
+  margin: 5px 0 0 13px; 
+}
+
+
+// Mobile navigation menu
+
+#mobile-nav 
+{
+  display: none;
+}
+
+@media screen and (max-width: 770px)
+{
+  #mobile-nav
   {
-    li
-    {
-      a
-      {
-        &:hover
-        {
-          text-decoration: underline;
-        }
-        text-decoration: none;
-        color: @reverse;
-      }
-      display: inline-block;
-      padding: 10px 10px 10px 0;
-    }
-    padding: 0 30px;
-    list-style: none;
-    max-width: 1000px;
-    margin: 0px auto;
+    display: inline-block;
```

Comment:
Suggestion/detail: We can also use `display: block;` here because we are not going to use features of `inline-block` here other, than it acting as a block level element.